src/test/java/net/helpscout/api/ApiClientTest.java

```diff
@@ -1,12 +1,26 @@
 package net.helpscout.api;
 
-import static com.github.tomakehurst.wiremock.client.WireMock.*;
```

Comment:
Not a huge deal, but I'd probably leave the various import * here if possible since we're using numerous items from those packages.